cmd/rollkit/commands/yaml.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	rollconf "github.com/rollkit/rollkit/config"
+)
+
+// NewYamlCmd creates a new cobra command group for YAML file operations.
+func NewYamlCmd() *cobra.Command {
+	yamlCmd := &cobra.Command{
+		Use:     "yaml",
```

Comment:
i wouldnt make this a sub command, we can just do init. we dont support other forms